client/scripts/index.js

```diff
@@ -53,59 +72,61 @@ function sketchpad_init(Sketchpad) {
     return pad
 }
 
-function game_init(pad) {
+function gameInit(pad) {
     let player_number = Math.floor(Math.random() * 10) // temporary placeholder for player number
-
     pad.setLineColor(getPlayerColorFromNumber(player_number))
-
-
     document.getElementById('submit_stroke').onclick = function() {
-        submit_stroke_to_server(player_number, pad)
+        submitStrokeToServer(player_number, pad)
     };
-
 }
 
 // after a player has submitted their stroke,
 // this function exports the whiteboard as JSON and sends it to the server
-function submit_stroke_to_server(player_number, pad) {
+function submitStrokeToServer(player_number, pad) {
```

Comment:
this function makes it sound like you are passing a stroke to the server
but the arg is pad
so either change the function to be
submit pad to server
OR
modify the param to be the strokeJson data instead of pad